airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/db/DbAnalyticsUtils.java

```diff
@@ -0,0 +1,15 @@
+package io.airbyte.cdk.db;
+
+import io.airbyte.protocol.models.v0.AirbyteAnalyticsTraceMessage;
+
+/**
+ * Utility class to define constants associated with database source connector analytics events. Make sure to add the analytics event to
+ * https://www.notion.so/Connector-Analytics-Events-892a79a49852465f8d59a18bd84c36de
+ */
+public class DbAnalyticsUtils {
+  public static final String CDC_CURSOR_INVALID_KEY = "db-sources-cdc-cursor-invalid";
+
+  public static AirbyteAnalyticsTraceMessage cdcCursorInvalidMessage() {
+    return new AirbyteAnalyticsTraceMessage().withType(CDC_CURSOR_INVALID_KEY).withValue("1");
```

Comment:
what does this value stand for?

Reply:
It's a metric that we have defined for invalid cdc cursor. It will eventually show up here : https://airbyte.metabaseapp.com/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7ImRhdGFiYXNlIjoyLCJxdWVyeSI6eyJzb3VyY2UtdGFibGUiOjMyOTUzfSwidHlwZSI6InF1ZXJ5In0sImRpc3BsYXkiOiJ0YWJsZSIsInZpc3VhbGl6YXRpb25fc2V0dGluZ3MiOnt9fQ==

We can build visualizations on top of it to understand connector analytics